packages/react-core/package.json

```diff
@@ -0,0 +1,123 @@
+{
+  "name": "@thirdweb-dev/react-core",
+  "version": "1.0.0",
```

Comment:
let's start this at `0.1.0`? - actually `0.0.0` so that the changesets can bump it to `0.1.0` on release